modules/globebrowsing/other/distanceswitch.cpp

```diff
@@ -46,44 +45,33 @@ bool DistanceSwitch::deinitialize() {
     return true;
 }
 
-
 void DistanceSwitch::render(const RenderData& data) {
-    if (_maxDistances.size() == 0) {
-        return;
-    }
-
-    double distanceToCamera = distance(data.camera.positionVec3(), data.modelTransform.translation);
-
-    if (distanceToCamera > _maxDistances.back() * _objectScale) {
-        return;
-    }
+    
+    double distanceToCamera =
+        distance(data.camera.positionVec3(), data.modelTransform.translation);
 
+    // This distance will be enough to render the globe as one pixel if the field of
+    // view is 'fov' radians and the screen resolution is 'res' pixels.
+    double fov = 2 * glm::pi<double>() / 6; // 60 degrees
+    int res = 2880;
```

Comment:
Because that was my retina screen resolution :P. Hehe. This is basically just a heuristic with a high value for the resolution. I didn't want to use the actual resolution since it may differ between screens causing different behaviour across edges.